mappings/net/minecraft/unmapped/BiomeColor.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/unmapped/C_qodopjcw net/minecraft/unmapped/BiomeColor
```

Comment:
This class is already mapped by #5, same for FogShape below

Reply:
okie i fixy time